lib/features/wallet_screen/add_contact_page/bloc/contacts_state.dart

```diff
@@ -0,0 +1,15 @@
+part of 'contacts_bloc.dart';
+
+abstract class ContactsState {
+  const ContactsState();
+}
+
+class ContactsInitial extends ContactsState {}
+
+class ContactsLoaded extends ContactsState {
```

Comment:
Do we actually need two states just for contacts? I think the state could be just `List<Contact>`. 

I made `HashesListState` the same way 1.5 years ago. The logic was that users sees the scan page first, so it's important to understand if the list of objects is empty or wasn't initialized. However, in the case of contacts, I don't see the need for such a structure, it only complicates the code